docs/docs/guides/Web3_Utils_module/Mastering_web3-utils.md

```diff
@@ -0,0 +1,237 @@
+# Mastering Utility Functions
+
+## Introduction
+In this guide, you'll learn about the different functions of the web3 utils package, it contains the methods to know how to generate random bytes in different formats, how to perform conversion between Hex values and numbers, hashing functions, addresses, packing padding and in the last part you will see how to compare block numbers.
+
+## Installation
+
+Install only the web3 package
+```bash
+npm i web3-utils
+```
+
+Or you can install the web3 library as well and then access web3.utils
+
+```bash
+npm i web3
+```
+
+## Imports
+
+There are three different ways to import utils package.
+
+### Import the entire web3 library
+
+```js
+//import web3 module
+const { Web3 } = require("web3"); 
```

Comment:
```suggestion
import Web3 = from "web3"; 
```
I suggest using typescript when creating examples, thats one of our strengths in v4!